### ISSUE TITLE
Split design for organizations into separate repo

### ISSUE THREAD

**Issue by velfundert** [roisalen/angular-frontend#25]
This is in many ways a different approach to issue #19: I think it would be a good idea to refactor the individual organization design-parts so that they are compatible with both angular and fluxible, and split them into a separate repo. 

This way we:
- won't have to rewrite the designfiles more than once
- won't have organizations that only have their design available in one of the frontends
- won't have non-critical code (design) committed to the same repo as critical code (the code for the frontend)

**Comment by torkilv** [comment]
I agree,specific css for each organisation should be possible to retrieve from any app

**Comment by velfundert** [comment]
There are two different approaches for the design:

- Creating minimal css-files that can do the necessary color-changes in any theme, typically by using CSS-class definitions. This requires less processing, and lets you easily drop new sheets into production. At the same time, it might be more difficult to make it work with all our different future frontends

- Using Less / SASS and compile. Requires processing, but is more flexible.

- Doing some JS-magic each time a user loads the page: Slower, don't know about any libraries atm.
